### PR TITLE
Create scan product whit filter

### LINKED ISSUE #36: Scan filter
Create dynamic filter for scan operations.

### PR DESCRIPTION
**Pull request by URBA22**
Close #36

### CODE REVIEW

File: src/functions/products/createProduct/index.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default {
                 path: 'products',
                 cors: true,
                 request: {
-                    schema: {
+                    schemas: {
```
Comment (samupd2013) [inline src/functions/products/createProduct/index.ts:14]: Why this change?

Comment (samupd2013) [inline src/functions/products/createProduct/index.ts:14]: Why this change?

Comment (URBA22) [reply src/functions/products/createProduct/index.ts:14]: I fixed two  compiler warnings. 
I changes `schema` in `schemas` because this warning: 
> Serverless: Deprecation warning: Starting with next major version, "http.request.schema" property will be replaced by "http.request.schemas" 

and I split `iamRoleStatements` because this other warning:
> Serverless: Deprecation warning: Starting with version 3.0.0, following property will be replaced: "provider.iamRoleStatements" -> "provider.iam.role.statements"

**Review by samupd2013 — APPROVED** [review]

**Review by Manuv46 — APPROVED** [review]
